src/node_crypto.cc

```diff
@@ -2097,6 +2099,42 @@ void SSLWrap<Base>::GetPeerCertificate(
 }
 
 
+template <class Base>
+void SSLWrap<Base>::GetFinished(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  Base* w;
+  ASSIGN_OR_RETURN_UNWRAP(&w, args.Holder());
+
+  char dummy[1];
```

Comment:
This probably needs a comment, explaining that `nullptr` can't be passed to `SSL_get_finished`.

Reply:
@indutny Indeed, using a `dummy` byte without a comment is not readily understandable. Comments added, please review.